startup.sh

```diff
@@ -141,16 +141,16 @@ function startContainers {
   case $startupType in
       uniflow)
       
-        export CONSTRAINT_HOSTNAME=${nodeName}
+        export CONSTRAINT_ID=${nodeID}
```

Comment:
This var should be renamed from CONTRAINT_ID to something more appropriate

Reply:
like: SWARM_MGR_NODE_ID or SWARM_UNIFLOW_NODE_ID or anythong more descriptive

same for TEMPLATE_ID

Reply:
and I guess TEMPLATE_ID is the same as UNICONFIG_ID ? which could also be renamed